pages/data-providers/[data-provider-id]/doi.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+
+import { withGlobalStore } from 'store'
+import Table from 'components/table'
+import ExportButton from 'components/export-button'
+import texts from 'texts/doi'
+import { Card } from 'design'
+
+// TODO: Remove once cards are in @oacore/design
+// eslint-disable-next-line
+import styles from './doi.css'
+
+const DepositDates = ({
+  className,
+  store,
+  tag: Tag = 'main',
+  ...restProps
+}) => (
+  <Tag className={[styles.container, className].join(' ')} {...restProps}>
+    <h1>DOI</h1>
+
+    <Card className={styles.cardOne} tag="section">
+      <h2>Lorem ipsum</h2>
+    </Card>
+
+    <Card className={styles.cardTwo} tag="section">
+      <h2>{texts.exporting.title}</h2>
+      <p>
+        {texts.exporting.description.render({
+          count: store.depositDates.depositDatesCount || '',
+        })}
+      </p>
+      <ExportButton
+        href={store.doi.doiUrl}
+        disabled={store.doi.isExportDisabled}
+      >
+        {texts.exporting.download}
+      </ExportButton>
+    </Card>
+
+    <Card className={styles.doiTable} tag="section">
+      <Table
+        key={store.dataProvider}
+        title="Browse DOI records"
+        pages={store.doi.doiRecords}
+        searchable
+      >
+        <Table.Column
+          id="repo_doi"
+          display="DOI"
+          order=""
+          getter={({ crossrefDoi, repoDoi }) =>
+            crossrefDoi && !repoDoi ? `${crossrefDoi} 🆕` : repoDoi
+          }
+          className={styles.doiColumn}
+        />
+        <Table.Column
+          id="title"
+          display="Title"
+          order=""
+          className={styles.titleColumn}
+        />
+        <Table.Column
+          id="authors"
+          display="Authors"
+          order=""
+          className={styles.authorsColumn}
+          getter={v => v.authors && v.authors.map(a => a.name).join(' ')}
+        />
+      </Table>
+    </Card>
+  </Tag>
+)
+
+export default withGlobalStore(DepositDates)
```

Comment:
Why cannot you just pass the `store` as a prop from the `App` and do not use this context at all?

Reply:
this decorator makes class observe, i.e. if the store would be taken from APP it wouldn't retrigger rerender when `store.depositDates.depositDatesCount`  changes. So either you add `@observe` decorator or use this function. Both approaches are almost identical.